### PR TITLE
Upate Windows installation to use kubectl

### PR DESCRIPTION
**Pull request by song-jiang**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
TBD
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

File: calico/getting-started/windows-calico/quickstart.md [hunk 1]
```diff
@@ -31,9 +31,12 @@ For Linux control nodes using {{site.prodname}} networking, strict affinity must
 This is required to prevent Linux nodes from borrowing IP addresses from Windows nodes:
 
 ```bash
-calicoctl ipam configure --strictaffinity=true
+kubectl patch ipamconfigurations default --type merge --patch='{"spec": {"strictAffinity": true}}'
 ```
 
+> **Note**: If the above command failed to find ipamconfigurations resource, you need to install Calico API server. Please refer to [installing the Calico API server]({{site.baseurl}}/maintenance/install-apiserver).
```
Comment (lmm) [inline calico/getting-started/windows-calico/quickstart.md:37]: nit:

> If the above command failed to find the ipamconfigurations resource, you need to install the Calico API server.

Comment (lmm) [reply calico/getting-started/windows-calico/quickstart.md:37]: Do we want to add this note in the other Windows docs pages too?

**Review by lmm — APPROVED** [review]
LGTM, just a nit and a question.

File: calico/getting-started/windows-calico/quickstart.md (hunk 1 above)
Comment (song-jiang) [reply calico/getting-started/windows-calico/quickstart.md:37]: We can but I just feel a bit redundant. The quick start guard is the page most of the users will land.

**Pull request merged**